adapter/src/main/kotlin/org/javacs/ktda/adapter/DAPConverter.kt

```diff
@@ -76,7 +76,7 @@ class DAPConverter(
 	}
 
 	fun toDAPBreakpoint(internalBreakpoint: InternalExceptionBreakpoint) = DAPBreakpoint().apply {
-		id = internalBreakpoint.id.toInt()
```

Comment:
This is a bug i ran into.
I'm not 100% sure why but the `internalBreakpoint.id` is always C or U (caught or uncaught) not a number.
The `ordinal` seems to be 0 for caught and 1 for uncaught.

Its a naive fix but it works, and I need it for my tests to work :angel: